suppress.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+    <suppress until="2024-09-01Z">
+        <notes><![CDATA[
+   file name: jackson-databind-2.15.2.jar
+
+   latest version of this package currently has this vulnerability, please update when new version comes out
```

Comment:
The CVE is disputed by the vendor. Could you put the link to the issue in the notes? https://github.com/FasterXML/jackson-databind/issues/3972

Reply:
done